docs/EN_US/ContainerizedHPCC/ContainerizedMods/ConfigureValues.xml

```diff
@@ -0,0 +1,1255 @@
+        (minimal for listening to the queue etc.) and all the jobs it may have
+        to run in parallel.</para>
+
+        <para>For example, imagine <emphasis>maxJobs</emphasis> is set to 4,
+        and 4 large queries are queued rapidly, that will mean 4 child
+        processes are launched each consuming cpu and memory within the
+        eclccserver pod. With the component configured with
+        <emphasis>useChildProcesses</emphasis> set to true, each job will run
+        in the same pod (up to the value of <emphasis>maxJobs</emphasis> in
+        parallel). Therefore with <emphasis>useChildProcesses</emphasis>
+        enabled, the component resources must be defined such that the pod has
+        enough resources to handle the resource demands of all the those jobs
+        to be able to run in parallel.</para>
+
+        <para>With useChildProcess enabled it could be rather expensive in
+        most cloud pricing models, and rather wasteful if there aren't any
+        jobs running. Instead you can set this
+        <emphasis>useChildprocess</emphasis> to false (the default) to start
+        your pod with only the required memory for the job which will be
+        disposed of when done. Now this model also has overheard, perhaps 20
+        seconds to a minute to spawn the Kubernetes cluster to process the
+        job. Which may not be ideal for an environment which is sending
+        several small jobs, but rather larger jobs which would minimize the
+        effect of the overhead in starting the Kubernetes cluster. Setting
+        <emphasis>useChildProcess</emphasis> to false better allows for the
+        possibility of dynamic scaling. For jobs which would take a long while
+        to compile, the extra (start up) overhead is minimal, and that would
+        be the ideal case to have the <emphasis>useChildProcess</emphasis> as
+        false. Setting <emphasis>useChildProcess</emphasis> to false only
+        allows 1 pod per compile.</para>
+
+        <para><emphasis role="bold">maxActive</emphasis>: The maximum number
+        of jobs that can be run in parallel. Again use caution because each
+        job will need enough memory to run. For instance, if
+        <emphasis>maxActive</emphasis> is set to 2000, you could submit a very
+        big job and in that case spawn some 2000 jobs using a considerable
+        amount of resources, which could potentially run up a rather expensive
+        compilation bill, again depending on your cloud provider and your
+        billing plan.</para>
+      </sect3>
+
+      <sect3 id="ValYML_Sasha">
+        <title>Sasha</title>
+
+        <para>Sasha is an outlier as it is a dictionary type structure and not
+        a list. You can't have more than one archiver or dfuwu-archiver as
+        that is a value limitation, you can choose to either have the service
+        or not (set the 'disabled' value to true). The Sasha component is an
```

Comment:
Not sure what this sentence is conveying.  Is it just a repeat of the 1st in the paragraph.

Reply:
likely. Just 'notes' I had jotted down felt relevant. If irrelevant or repetitive, should I remove??